src/pages/Technical/Technical.jsx

```diff
@@ -0,0 +1,109 @@
+import React, { useState, useEffect } from 'react'
+import { Alert, Col, Row } from 'react-bootstrap'
+import { ArrowLeft } from 'react-bootstrap-icons'
+import Page from '@/components/Page/Page'
+import ButtonCard from './components/ButtonCard'
+import alert from '@/utils/alert'
+import TechnicalForm from '@/forms/Techical'
+import TechnicalApi from '@/API/technical'
+// import OttApi from '@/API/ott'
+
+import electrician from './icons/electrician.png'
+import plumber from './icons/plumber.png'
+import carpenter from './icons/carpenter.png'
+
+export default function Technical() {
+    const [service, selectService] = useState(null)
+
+    const isServiceSelected = service !== null
+
+    useEffect(() => window.scrollTo(0, 0), [service])
+
+    function backToSelection() {
+        selectService(null)
+    }
+
+    async function sendRequest(values) {
+        // if (
+        //     await TechnicalApi.post(
+        //         { text: values.text, category: service },
+        //         await OttApi.getToken({
+        //             email: values.email,
+        //             surname: values.surname,
+        //             name: values.name,
+        //             patronymic: values.patronymic,
+        //             building: values.building,
+        //             room: values.room,
+        //         })
+        //     )
+        // ) {
+        //     backToSelection()
+        //     alert(
+        //         'Вы успешно подали заявку на оказание технических услуг, ожидайте прихода мастера',
+        //         {
+        //             title: 'Успех!',
+        //         }
+        //     )
+        // }
+    }
+
+    return (
+        <Page header="Электронная запись на оказание технических услуг">
+            {!isServiceSelected ? (
+                <>
+                    <Alert variant="info">
+                        На этой странице можно оставить заявку на оказание технических услуг. Для
+                        этого нужно выбрать желаемую категорию услуги, после заполнить контактную
+                        информацию и оставить описание проблемы. Далее останется лишь дождаться
+                        прихода мастера.
+                    </Alert>
+                    <Alert variant="warning">
+                        Заявки с этой страницы отправляются каждый день в 8 часов утра. Если вам
+                        срочно нужна помощь - обратитесь к коменданту.
+                    </Alert>
+                    <Row className="gx-5 gy-3 mt-3 mb-3 justify-content-center">
```

Comment:
Снова докапываюсь до отступов:
```suggestion
                    <Row className="gx-5 gy-3 gy-lg-0 justify-content-center">
```
`mt-3` - не работает, ибо у элемента сверху есть mb-3
`mb-3` - не нужно, ибо элемент в конце страницы, у которой уже по умолчанию есть внутеренний паддинг

`gy-3` добавляет лишние маржины колонкам внутри `<Row>`, и он имеет смысл только на мобильных разрешениях. Поэтому добавил `gy-lg-0`

Так выглядит более аккуратно